apps/explorer/src/app/routes/txs/home/index.tsx

```diff
@@ -119,4 +119,12 @@ const Wrapper = () => {
   );
 };
 
-export { Wrapper as TxsHome };
+export const TxsHomeFallback = () => (
+  <>
+    <RouteTitle>{t('Transactions')}</RouteTitle>
+    <div>
+      The transactions list is currently disabled. Please use the search bar to
+      discover transaction data
```

Comment:
follow up (non-blocking): 
```suggestion
      t('The transactions list is currently disabled. Please use the search bar to
      discover transaction data.')
```